Dockerfile-builder

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:16.04
+
+RUN apt-get update \
+ && apt-get install -y \
+    git build-essential software-properties-common sudo \
+ && add-apt-repository -y ppa:ubuntu-toolchain-r/test \
+ && apt-get update \
+ && apt-get install -y gcc-7 \
+ && cd /usr/bin/ \
+ && rm gcc \
+ && ln -s gcc-7 gcc \
+ && git clone https://gerrit.fd.io/r/vpp /vpp
+
+ADD build.env /
+
+RUN . /build.env \
+ && cd /vpp \
+ && git pull \
+ && git checkout ${VPP_COMMIT} \
+ && UNATTENDED=y make vpp_configure_args_vpp='--disable-japi --disable-vom --disable-papi' install-dep bootstrap pkg-deb \
```

Comment:
did you mean to use VPP_BUILD_ARGS here?

Reply:
`sh` looks doesn't like it .. I'll remove it from build.env